testgrid/config/config.yaml

```diff
@@ -1986,6 +1986,29 @@ dashboards:
   - name: aws-janitor
     test_group_name: maintenance-ci-aws-janitor
 
+- name: submit queue blocking
```

Comment:
Please don't be so verbose and also don't add spaces. So how about `blocking` or `sq-blocking`?